spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -23,31 +23,31 @@ You also need to have link:http://redis.io/[Redis] installed and running if you
 === Deploying 'local'
 
 [start=1]
-1. download the Spring Cloud Data Flow Admin and Shell apps:
+1. download the Spring Cloud Data Flow Server and Shell apps:
 
 ```
-wget http://repo.spring.io/milestone/org/springframework/cloud/spring-cloud-dataflow-admin-local/1.0.0.M2/spring-cloud-dataflow-admin-local-1.0.0.M2.jar
-wget http://repo.spring.io/milestone/org/springframework/cloud/spring-cloud-dataflow-shell/1.0.0.M2/spring-cloud-dataflow-shell-1.0.0.M2.jar
+wget http://repo.spring.io/milestone/org/springframework/cloud/spring-cloud-dataflow-server-local/1.0.0.BUILD-SNAPSHOT/spring-cloud-dataflow-server-local-1.0.0.BUILD-SNAPSHOT.jar
```

Comment:
Maybe not part of this PR, but it'd be so much better to extract the version in an asciidoc attribute